remill/BC/DeadStoreEliminator.cpp

```diff
@@ -1120,7 +1133,7 @@ VisitResult ForwardAliasVisitor::visitPHINode(llvm::PHINode &inst) {
 }
 
 VisitResult ForwardAliasVisitor::visitCallInst(llvm::CallInst &inst) {
-  const auto val = inst.getCalledValue()->stripPointerCasts();
+  const auto val = inst.getCalledFunction()->stripPointerCasts();
```

Comment:
`getCalledFunction` might return `nullptr`.

Reply:
updated to `getCalledOperand()`